src/Polkadot/Extrinsic.cpp

```diff
@@ -79,6 +80,21 @@ bool Extrinsic::encodeRawAccount() const {
     return true;
 }
 
+Data Extrinsic::encodeStakingController(const Proto::Staking::Bond& bond) const {
+    if (bond.controller().empty()) {
+        return {};
+    }
+
+    bool polkadotOrKusama = network == static_cast<uint32_t>(TWSS58AddressTypePolkadot)
+                            || network == static_cast<uint32_t>(TWSS58AddressTypeKusama);
+    if (polkadotOrKusama && specVersion >= stakingControllerSpecVersion) {
```

Comment:
What's the point of checking `specVersion`? There's no way of going back :)

Reply:
I've rolled back these changes, thank you for the note!
I was thinking about this yesterday, and came to the decision that it is better to do version checking on the end-user app side